src/fragments/lib/logging/common/sending_logs/sending_logs.mdx

```diff
@@ -0,0 +1,20 @@
+The Amplify library already uses the Amplify Logger internally.  Each Amplify feature logs messages at various log levels.  You can log your own custom messages and various log levels depending on your use cases.  Only log levels that are the same or below the log level defined in the logging constraints are sent to Amazon CloudWatch.
```

Comment:
With the Amplify Logger, you can send logs to Amazon CloudWatch from errors that are caught by the Amplify library or by adding your own custom log messages. For the Amplify library and custom logs you can customize which levels to log to CloudWatch.

Reply:
Text updated per suggestion